Spigot-Server-Patches/0348-Cached-IBlockData-to-CraftBlockData-conversion.patch

```diff
@@ -0,0 +1,76 @@
+From e64041afdb1a16feb924eb453e26db2ff7c36438 Mon Sep 17 00:00:00 2001
+From: miclebrick <miclebrick@outlook.com>
+Date: Thu, 23 Aug 2018 11:45:32 -0400
+Subject: [PATCH] Cached IBlockData to CraftBlockData conversion
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockData.java b/src/main/java/net/minecraft/server/BlockData.java
+index 4dd6c3276..71a2f6dc2 100644
+--- a/src/main/java/net/minecraft/server/BlockData.java
++++ b/src/main/java/net/minecraft/server/BlockData.java
+@@ -1,6 +1,7 @@
+ package net.minecraft.server;
+ 
+ import com.google.common.collect.ImmutableMap;
++import org.bukkit.craftbukkit.block.data.CraftBlockData;
+ 
+ public class BlockData extends BlockDataAbstract<Block, IBlockData> implements IBlockData {
+ 
+@@ -11,4 +12,14 @@ public class BlockData extends BlockDataAbstract<Block, IBlockData> implements I
+     public Block getBlock() {
+         return (Block) this.e_;
+     }
++
++    // Paper start - impl cached craft block data, lazy load to fix issue with loading at the wrong time
++    private CraftBlockData cachedCraftBlockData;
```

Comment:
Just go ahead and create it on this line. We don't need that iteration in CraftBlockData